### DISCUSSION TITLE
How to make it work for https

### DISCUSSION THREAD

**Discussion by xKiian** [synchronizing/mitm#26] [Q&A] [answered]
for me, mitm just works with http packets and not https.

do i need a certificate or something? 

thanks in advice :D

**Comment by synchronizing** [comment, marked as answer]
Hey. Yes, you would need to install the certificates. See [Trusting `mitm`](https://synchronizing.github.io/mitm/docs/trusting-mitm.html) in the docs for more detail. Once the CA cert is trusted on your system, you'll be able to intercept and see `https` traffic.

**Comment by xKiian** [comment]
ok so i installed the certificate, restartet my pc, but i still cant use https with it D:

**Reply by synchronizing** [reply]
You need to write more details so I can help out. Here are a few things to include:

* Code you are using to generate the CA certificates, load them to `mitm`, and start the server.
* How you are installing the certificates on your machine.
* The error/issue you are having.
